src/graph_export/graph_cli_parser.cpp

```diff
@@ -29,69 +29,65 @@
 
 namespace ovms {
 
-GraphSettingsImpl& GraphCLIParser::defaultGraphSettings() {
-    static GraphSettingsImpl instance;
+TextGenGraphSettingsImpl& GraphCLIParser::defaultGraphSettings() {
```

Comment:
Its not only TextGen, it will be used for VLMs image2image text2image etc.

GenAIGraphSettings or leave with GraphSettings?